### DISCUSSION TITLE
Curation Report - Week of July 25

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/GPML-Repo#83] [Curation reports]
This week, there were 87 changes, including:
    - some changes to lipidmaps pathways updating IDs
    - most changes are adding clickable pathways by Egon

12 pathways under construction:
    - I removed the under construction tag and added the incomplete tag for https://www.wikipathways.org/index.php/Pathway:WP5237 as the last changes were end of May. @egonw are you still working on this?
- there are some covid related pathways created on the 5th of July, maybe at a workshop? Are there plans to finish them?

Incomplete pathways:

- I've added the request deletion tag for https://www.wikipathways.org/index.php/Pathway:WP1500. Reason: This pathway has not seen major work since 2010. All nodes and connections are also included here https://www.wikipathways.org/index.php/Pathway:WP1498.

- In https://www.wikipathways.org/index.php/Pathway:WP5091 I was curious what "frustrated phagocytosis" was, and found that the aopwiki entry is missing the c in the title, but I don't know how to change it there. @marvinm2 

Search works.
Next one is @fehrhart

**Comment by fehrhart** [comment]
Dear all,

  my report of this week. A lot of changes to check, but nothing unusual -
special thanks to Egon for making a lot of pathways in pathways clickable
:)

I approved WP5233 - Arsenic metabolism (from Zuzanna, Marvins student)

Marvin what about some of the AOP pathways without molecular information?
Is it OK to add them to the AOP community?
https://www.wikipathways.org/index.php/Pathway:WP4017
https://www.wikipathways.org/index.php/Pathway:WP5228
https://www.wikipathways.org/index.php/Pathway:WP5229

Denise, what is happening with these ones?
https://www.wikipathways.org/index.php/Pathway:WP4463
https://www.wikipathways.org/index.php/Pathway:WP4161
They need a lot of node annotation but look complete apart from that.

I furthermore added  these ones:
https://www.wikipathways.org/index.php/Pathway:WP5257
...
https://www.wikipathways.org/index.php/Pathway:WP5263
from the recent COVID19 workshop to the COVID-19 community.

Next one up is Nhung!

Best regards,
Freddie
